tests/unit/test_session.py

```diff
@@ -407,11 +407,21 @@ class SubModel(base_model):
 
     expected = {
         'AttributeDefinitions': [
-            {'AttributeName': 'id', 'AttributeType': 'S'}],
+            {'AttributeName': 'email', 'AttributeType': 'S'},
```

Comment:
The comment on 404 above this is out of date, since it now includes the parent's attributes as well.